.github/workflows/ci.yml

```
@@ -68,3 +68,37 @@
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
           # echo "image=$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG" >> $GITHUB_OUTPUT
+
+      - name : Checkout k8s repository
+        uses : actions/checkout@v4
+        with:
+          repository : GCU-Home-Protector/home-protector-infra
+          ref : main
+          token : ${{secrets.ACTION_TOKEN}}
+          path : home-protector-infra
+
+      - name : Update backend image
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)